src/test/java/com/uwetrottmann/tmdb/services/TvServiceTest.java

```diff
@@ -13,38 +24,196 @@
+        assertThat(images.backdrops.get(0).vote_count).isPositive();
+        assertThat(images.posters).isNotEmpty();
+        assertThat(images.posters.get(0).file_path).isNotNull();
+        assertThat(images.posters.get(0).width).isEqualTo(1000);
+        assertThat(images.posters.get(0).height).isEqualTo(1500);
+        assertThat(images.posters.get(0).iso_639_1).isEqualTo("en");
+        assertThat(images.posters.get(0).aspect_ratio).isGreaterThan(0.6f);
+        assertThat(images.posters.get(0).vote_average).isPositive();
+        assertThat(images.posters.get(0).vote_count).isPositive();
+    }
+    
+    @Test
+    public void test_keywords() {
+        TvKeywords keywords = getManager().tvService().keywords(TestData.TVSHOW_ID);
+        assertThat(keywords).isNotNull();
+        assertThat(keywords.id).isEqualTo(TestData.TVSHOW_ID);
+        assertThat(keywords.results.get(0).id).isNotNull();
+        assertThat(keywords.results.get(0).name).isNotNull();
+    }
+    
+    @Test
+    public void test_similar() {
+        TvResultsPage results = getManager().tvService().similar(TestData.TVSHOW_ID, 1, null);
+        assertThat(results).isNotNull();
+        assertThat(results.page).isNotNull().isPositive();
+        assertThat(results.total_pages).isNotNull().isPositive();
+        assertThat(results.total_results).isNotNull().isPositive();
+        assertThat(results.results).isNotEmpty();
+        assertThat(results.results.get(0).backdrop_path).isNotNull();
+        assertThat(results.results.get(0).id).isNotNull().isPositive();
+        assertThat(results.results.get(0).original_name).isNotNull();
+        assertThat(results.results.get(0).first_air_date).isNotNull();
+        assertThat(results.results.get(0).poster_path).isNotNull();
+        assertThat(results.results.get(0).popularity).isNotNull().isPositive();
+        assertThat(results.results.get(0).name).isNotNull();
+        assertThat(results.results.get(0).vote_average).isNotNull().isPositive();
+        assertThat(results.results.get(0).vote_count).isNotNull().isPositive();
+    }
+    
+    @Test
+    public void test_videos() {
+        Videos videos = getManager().tvService().videos(TestData.TVSHOW_ID, null);
+        assertThat(videos).isNotNull();
+        assertThat(videos.id).isEqualTo(TestData.TVSHOW_ID);
+        assertThat(videos.results.get(0).id).isNotNull();
+        assertThat(videos.results.get(0).iso_639_1).isNotNull();
+        assertThat(videos.results.get(0).key).isNotNull();
+        assertThat(videos.results.get(0).name).isNotNull();
+        assertThat(videos.results.get(0).site).isEqualTo("YouTube");
+        assertThat(videos.results.get(0).size).isNotNull();
+        assertThat(videos.results.get(0).type).isEqualTo("Opening Credits");
+    }
+    
+    @Test
+    public void test_latest() {
+        TvShowComplete show = getManager().tvService().latest();
+        assertTvShow(show);
```

Comment:
Can't be sure what is returned there (e.g. `poster_path` or other non-essential stuff is `null`), so probably best if you just test for the essential things (like there is a TMDB id).